packages/@apphosting/adapter-nextjs/src/overrides.ts

```diff
@@ -10,26 +10,38 @@ import {
 import { join, extname } from "path";
 import { rename as renamePromise } from "fs/promises";
 
+const DEFAULT_NEXT_CONFIG_FILE = "next.config.js";
+
 /**
  * Overrides the user's Next Config file (next.config.[ts|js|mjs]) to add configs
  * optimized for Firebase App Hosting.
  */
 export async function overrideNextConfig(projectRoot: string, nextConfigFileName: string) {
   console.log(`Overriding Next Config to add configs optmized for Firebase App Hosting`);
-  // Check if the file exists in the current working directory
-  const configPath = join(projectRoot, nextConfigFileName);
-
-  if (!(await exists(configPath))) {
-    console.log(`No Next.js config file found at ${configPath}`);
+  const userNextConfigExists = await exists(join(projectRoot, nextConfigFileName));
+  if (!userNextConfigExists) {
+    console.log(`No Next config file found, creating one with Firebase App Hosting overrides`);
+    try {
+      await writeFile(join(projectRoot, DEFAULT_NEXT_CONFIG_FILE), defaultNextConfigForFAH());
```

Comment:
Does `nextConfigFileName` include the default filename? Then we can drop `nextConfigFileName` in favor.